docs/_articles/en/soql/soql-builder.md

```diff
@@ -60,6 +61,8 @@ As you build your query, watch SOQL Builder display the query syntax while it si
 
 You can select objects and fields from the drop-down list, or type to narrow the list results. You can select an object or a field only once. If a value is already selected, it doesn't appear in the drop-down or search results.
 
+If building a query using COUNT(), all other selected fields are removed. If you didn't intend to select COUNT, you can undo the action from the main menu. You can further refine the results by adding filters (WHERE clauses). When you run the query, the number of returned rows corresponds to the total number of records. <!-- ekapner, 2/12: new doc -->
```

Comment:
Minor suggested edit: "When building a query using COUNT, all other selected fields are removed.  If you didn't intend to select COUNT, undo the action from the main menu.  Further refine the results by adding filters (WHERE clauses)."

Question: It might be my lack of context, but I'm a tad confused by "the number of returned rows corresponds to the total number of records."? Total number of records where?  In the sObject?

Reply:
In Query Results window. I'm going to Slack you a screen shot. Maybe I need to add one until we improve the UI experience, which "we" plan to do.

Reply:
If you're trying to explain that COUNT returns, I think it's fine to just remove the sentence.  You explain above what count means, and I reckon it's self-explanatory.  But again, I don't see the context so maybe I"m jsut confused :)

Reply:
Just to finish up, keeping your wording and adding an explanatory image is perfect. Phew!